jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -303,6 +303,8 @@ class recursion_set_entryt
   }
 };
 
+const std::string printable_char_set = " -~";
```

Comment:
:no_entry_sign: Please document this and maybe give it a better name, I would expect a printable_char_set to include, for example, `'a'`! 

:pick: I also would expect a set to be a `std::unordered_set<char>`

Reply:
The range of printable characters if from '  ' to '~'